diff --git a/config b/config
index 453a45e..bead08d 100644
--- a/config
+++ b/config
@@ -14,9 +14,9 @@
 # IPv4 syntax: address:port
 # IPv6 syntax: [address]:port
 # IPv6 adresses are configured to only allow IPv6 connections
-hosts = 0.0.0.0:5232
+hosts = 127.0.0.1:5232
 # Daemon flag
-daemon = False
+daemon = True
 # File storing the PID in daemon mode
 pid =
 # SSL flag, enable HTTPS protocol
@@ -100,7 +100,7 @@ http_password_parameter =
 type = None
 
 # File for rights management from_file
-file = ~/.config/radicale/rights
+file = /var/lib/radicale/rights
 
 
 [storage]
@@ -109,7 +109,7 @@ file = ~/.config/radicale/rights
 type = filesystem
 
 # Folder for storing local collections, created if not present
-filesystem_folder = ~/.config/radicale/collections
+filesystem_folder = /var/lib/radicale/collections
 
 # Database URL for SQLAlchemy
 # dialect+driver://user:password@host/dbname[?key=value..]
diff --git a/logging b/logging
index f4efb88..b10f88c 100644
--- a/logging
+++ b/logging
@@ -45,7 +45,7 @@ formatter = simple
 [handler_file]
 # File handler
 class = FileHandler
-args = ('/var/log/radicale',)
+args = ('/var/log/radicale/radicale.log',)
 formatter = full
 
 
